### PR TITLE
[Improve][Connector-V2] Doris stream load use FE instead of BE

### PR DESCRIPTION
**Pull request by Hisoka-X**
### Purpose of this pull request
This PR change doris stream load endpoint from BE to FE. Before this PR, we randomly selected BE, and it would be better to let Doris FE judge the ability to select BE.

### Does this PR introduce _any_ user-facing change?
no

### How was this patch tested?
exist test.

### Check list

* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
* [ ] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

File: seatunnel-connectors-v2/connector-doris/src/main/java/org/apache/seatunnel/connectors/doris/sink/writer/DorisSinkWriter.java [hunk 1]
```diff
@@ -105,9 +98,8 @@ public DorisSinkWriter(
         this.initializeLoad();
     }
 
-    private void initializeLoad() throws IOException {
-        this.backends = RestService.getBackendsV2(dorisConfig, log);
-        String backend = getAvailableBackend();
+    private void initializeLoad() {
+        String backend = dorisConfig.getFrontends();
```
Comment (TaoZex) [inline seatunnel-connectors-v2/connector-doris/src/main/java/org/apache/seatunnel/connectors/doris/sink/writer/DorisSinkWriter.java:102]: It looks like it just returns fe. I wonder what's the purpose here?

Comment (Hisoka-X) [reply seatunnel-connectors-v2/connector-doris/src/main/java/org/apache/seatunnel/connectors/doris/sink/writer/DorisSinkWriter.java:102]: This is the purpose of this PR. Change stream load endpoint from BE to FE.

Comment (TaoZex) [reply seatunnel-connectors-v2/connector-doris/src/main/java/org/apache/seatunnel/connectors/doris/sink/writer/DorisSinkWriter.java:102]: Fe is mainly responsible for doris's query plan. I am worried that if stream load also selects fe, the efficiency of fe will be affected in high concurrency scenarios.

Comment (Hisoka-X) [reply seatunnel-connectors-v2/connector-doris/src/main/java/org/apache/seatunnel/connectors/doris/sink/writer/DorisSinkWriter.java:102]: Never mind. FE will use http redirect, then data also send from SeaTunnel to BE.
image

Comment (TaoZex) [reply seatunnel-connectors-v2/connector-doris/src/main/java/org/apache/seatunnel/connectors/doris/sink/writer/DorisSinkWriter.java:102]: Thanks for your response.

**Review by TaoZex — APPROVED** [review]
+1

**Review by EricJoy2048 — APPROVED** [review]

**Review by hailin0 — APPROVED** [review]

**Pull request merged**